src/main/js/TemplateInfo.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC } from "react";
+import { Notification } from "@scm-manager/ui-components";
+import { useTranslation } from "react-i18next";
+
+type Props = {
+  path?: string;
+};
+
+const TemplateInfo: FC<Props> = ({ path }) => {
+  const [t, i18n] = useTranslation("plugins");
+
+  const language = i18n.language === "de" ? "de" : "en";
+  const documentationUrl = `https://www.scm-manager.org/plugins/scm-repository-template-plugin/docs/1.0.x/${language}/usage/`;
```

Comment:
Please remove the `www` our official url is `scm-manager.org` the `www` is only for backwards compatibility.